propel/schema.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<database name="pizzaService">
+    <table name="ingredients" phpName="Ingredient">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true" />
+    </table>
+
+    <table name="pizzas" phpName="Pizza">
+        <column name="id" type="integer" required="true" primaryKey="true" autoIncrement="true" />
+        <column name="name" type="varchar" size="120" required="true"/>
+        <column name="price" type="float" required="true"/>
+    </table>
+
+    <table name="pizza_ingredients" phpName="Ingredient_pizza" isCrossRef="true">
+        <column name="ingredients_id" type="integer" primaryKey="true" required="true" />
+        <column name="pizzas_id" type="integer" primaryKey="true" required="true" />
```

Comment:
Please change the columns to not have plural form when referenced from a foreign table.
(It is only the one ingredient or pizza id, so using plural form does not make sense).
So the correct name would be `ingredient_id` and `pizza_id` etc.
Please change that on all tables!
Please also take into account that you may need to modify your commands that they use the new column names if necessary.